repo/packages/K/kubernetes/1/marathon.json

```diff
@@ -2,6 +2,7 @@
   "id": "{{kubernetes.framework-name}}",
   "cpus": {{kubernetes.cpus}},
   "mem": {{kubernetes.mem}},
+  "acceptedResourceRoles": {{kubernetes.resource-roles}},
```

Comment:
if we're *always* going to include this marathon field, then this `resource-roles` property should be listed in the required section.